assets/src/edit-story/app/story/useStoryReducer/reducers/arrangeElement.js

```diff
@@ -48,10 +48,10 @@ import { getAbsolutePosition, moveArrayElement } from './utils';
  * @param {Object} state Current state
  * @param {Object} payload Action payload
  * @param {string} payload.elementId Id of element to move
- * @param {number} payload.position New position of element to move
+ * @param {number|function(number):number} payload.position New position of element to move
  * @return {Object} New state
  */
-function arrangeElement(state, { elementId, position }) {
+function arrangeElement(state, { elementId, position: positionOrUpdater }) {
```

Comment:
It's for multi-element update. I did the same for `updateElements` and I think that works fairly well universally: either provide a new state or a calculation function. But I like your magic values too. I'll go ahead and use them since they make it more like an action.

Reply:
Done